src/bindings/python/flux/core/handle.py

```diff
@@ -80,6 +81,24 @@ def rpc_create(self, topic,
         """ Create a new RPC object """
         return RPC(self, topic, payload, nodeid, flags)
 
+    def mrpc_send(self, topic,
+                  payload=None,
+                  rankset="any",
+                  flags=0):
+        """
+        Create and send an MRPC in one step.
+        Returns a list of response tuples: (nodeid, payload)
+        """
+        with MRPC(self, topic, payload, rankset, flags) as mrpc:
+            return list(mrpc)
```

Comment:
> Is the iteration necessary to send it to all destinations?

No, I'm pretty sure it gets sent out when the `MRPC` is created.  So yeah, keeping it as an iterable makes sense.  I was going to hold off on implementing @grondo's suggestion of `f.rpc`, `f.event`, and `f.mrpc` until later, but maybe it makes sense to do dump `mrpc_create` and `mrpc_send` (since I don't see any difference between the two) for `f.mrpc` now.  Thoughts?

Reply:
Maybe just dump the mrpc_send one? Since the other seems equivalent.